frontend/src/pages/Landing/landing.module.css

```diff
@@ -30,6 +30,31 @@
   justify-content: center;
   align-items: center;
   padding: 16px;
+  position: relative;
+}
+
+.dti {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  bottom: 21px;
```

Comment:
typically we using rem for measurements to make sure things scale with the browser font size